<ui version="4.0" >
 <class>FindShows</class>
 <widget class="QWidget" name="FindShows" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Find Shows</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout_1" >
     <item>
      <widget class="QLabel" name="lblSearch" >
       <property name="text" >
        <string>&amp;Search:</string>
       </property>
       <property name="buddy" >
        <cstring>leSearch</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="imgProgress" >
       <property name="pixmap" >
        <pixmap resource="../../../pixmaps/nextshows.qrc" >:/pixmaps/prefs/progress_idle.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="NSLineEdit" name="leSearch" />
     </item>
     <item>
      <widget class="QPushButton" name="btnLookup" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Lookup</string>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
       <property name="flat" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible" >
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QLabel" name="lblSearchResultsTitle" >
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>Search results:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="NSTreeWidget" name="treeSearchResults" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="headerHidden" >
          <bool>true</bool>
         </property>
         <property name="columnCount" >
          <number>1</number>
         </property>
         <column>
          <property name="text" >
           <string/>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="QCheckBox" name="cbHideEndedShows" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="toolTip" >
            <string>Check this to filter out ended shows</string>
           </property>
           <property name="text" >
            <string>&amp;Hide ended</string>
           </property>
           <property name="checked" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddShow" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Add Show</string>
           </property>
           <property name="icon" >
            <iconset resource="../../../pixmaps/nextshows.qrc" >
             <normaloff>:/pixmaps/icons/rightarrow.png</normaloff>:/pixmaps/icons/rightarrow.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lblDisplayed" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="text" >
            <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&amp;nbsp;&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="" >
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QLabel" name="lblTrackedShowsTitle" >
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>Tracked shows:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="NSListWidget" name="lstTrackedShows" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="editTriggers" >
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRemoveShow" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Delete Show</string>
         </property>
         <property name="icon" >
          <iconset resource="../../../pixmaps/nextshows.qrc" >
           <normaloff>:/pixmaps/icons/trash.png</normaloff>:/pixmaps/icons/trash.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lblTrackedShows" >
           <property name="text" >
            <string/>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NSLineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/nslineedit.h</header>
  </customwidget>
  <customwidget>
   <class>NSTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>widgets/nstreewidget.h</header>
  </customwidget>
  <customwidget>
   <class>NSListWidget</class>
   <extends>QListWidget</extends>
   <header>widgets/nslistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../pixmaps/nextshows.qrc" />
 </resources>
 <connections/>
</ui>
